frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import PropTypes from "prop-types";
 import { t } from "ttag";
 
+import CheckBox from "metabase/core/components/CheckBox";
+import { EntityIconWrapper } from "metabase/components/EntityItem.styled";
```

Comment:
We should avoid importing from other components' `.styled` files. Ideally, we should treat them as component's internals. In that case, it could make sense to make `EntityIconWrapper` an independent component in `metabase/components`

Reply:
That makes sense, I can refactor it